CONTRIBUTING.md

```diff
@@ -169,13 +169,11 @@ It is expected that you will follow the GitFlow Workflow for managing the reposi
 * On completion it should be merged **not rebased** with development
 * You may see us creating `release/<version>` branches - this is where final testing will occur.
 * Where an issue is **assigned** to somebody it means they're working on it.  Speak to them before trying to contribute.
-* Where an issue is marked as **acknowledged** then it's been agreed that it is required and can be produced.
- * We would advise against working on issues without an Acknowledged label.
- * If you want to start work on an issue, comment on the issue asking to work on it and it'll be assigned to you by a developer.
+* If you want to work on an issue. Comment on the issue asking to work on it and a member of the team will add you to the project as a Contributor. This will then allow you to work on the ticket and update it as you go.
 * Any code change **must** have an associated issue detailing (in full) why the change is being made
-* Commit messages **must** make use of the issue management commands
- * When resolving an issue via a commit `Fixes #123 - Added the user's profile picture to their profile.`
- * When addressing an issue via a commit `Restructured the user's profile page to make room for the badges in #456`
+* Commit messages **must** reference the ticket and we **recommend** using the smart commit commands available to update your JIRA issue as described [here](https://confluence.atlassian.com/bitbucket/processing-jira-software-issues-with-smart-commit-messages-298979931.html)
```

Comment:
I am hesitant to say commit messages **must** reference the ticket, since this will clutter the GitHub commit messages with info that is likely unnecessary. Especially if we have a large feature, we wouldn't want every commit to reference an issue.

Reply:
Okay - fair enough. I agree that **must** is a bit too far, I think the option should remain however, for example in the case that someone is working on a feature, they should reference the feature's associated "Epic" issue in JIRA when creating the branch and PR. If individual issues within said epic are addressed in a particular commit, I would like to see them there.

Reply:
Thinking on this actually, commit-level smart functions such as issue links and comments do not actually work against our current workflow. As the repo is currently forked and merged back into, it would appear that only the branches and PRs are picked up by JIRA's link to the Core Repository as that is the point that code starts to be moved back into the Core Repo.